packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md

```diff
@@ -19,3 +19,31 @@ Similarly, when you enable a new capability in your application (for example, ad
     At this point, the Okta OIN team is notified and your submission undergoes the same process flow as the original submission.
 
     After the new version of the integration has reached the **Publish** stage and been published by Okta, the new version replaces the old one in the OIN catalog.
+
+>**Note:** You can have a maximum of 10 submissions for any development org in the OIN Manager.
+
+## Delete draft submissions
+
+There are two scenarios where you need to delete a draft submission:
+
+1. You have 10 draft submissions, which is the maximum number permitted in the OIN Manager.
+1. You have decided against completing a draft submission and want to remove it.
+
+In either of these scenarios, the OIN Manager provides a method to delete unpublished submissions. For instructions on how to delete app integrations that are already published in the OIN catalog, see [Delete published submissions](#delete-published-submissions).
+
+You can only delete unpublished submissions that are in **DRAFT** state. If the delete icon is dimmed, that means the delete operation isn't available for that submission. If you can't move your submission to the **DRAFT** state, contact the Okta OIN team at <oin@okta.com> for assistance.
```

Comment:
do we use the word "dimmed" in other places, over "inactive" or "disabled" referring to buttons that exist but cannot be clicked? nbd, as long as its consistent.

Reply:
Yeah - according to our style guide, which references https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/g/gray-grayed-out, when referring to the appearance of an icon, we should use "dimmed".

Reply:
I am not uber fond of "dimmed". Graham and I are actually currently considering "unavailable" as our official use and straying from Microsoft on this one.  You could then shorten that sentence.....

Reply:
Cleaned up the text to remove dimmed and tidy up the phrasing.